README.md

```diff
@@ -429,6 +429,7 @@ Be sure to replace the **VERSION** key below with the one of the versions shown
 **Gradle**
 ```gradle
 dependencies {
+    //Change 'compile' to 'implementation' in Gradle 7.0+
     compile 'net.dv8tion:JDA:VERSION'
```

Comment:
```suggestion
    //Change 'implementatoion' to 'compile' in old Gradle versions
    implementation("net.dv8tion:JDA:VERSION")
```

Reply:
implementatoion is a typo

Reply:
Then fix it locally?
```suggestion
    //Change 'implementation' to 'compile' in old Gradle versions
    implementation("net.dv8tion:JDA:VERSION")
```

Reply:
i'm fairly new to PRing into other repos so was unsure about how to do that, fixed it now.